embulk-standards/src/main/java/org/embulk/standards/LocalFileInputPlugin.java

```diff
@@ -103,17 +105,44 @@ public void cleanup(TaskSource taskSource,
             int taskCount,
             List<TaskReport> successTaskReports) {}
 
+    String escapeGlobPattern(String filenamePrefix) {
+        StringBuilder sb = new StringBuilder();
+        final int len = filenamePrefix.length();
+        for (int i = 0; i < len; i++) {
+            char c = filenamePrefix.charAt(i);
+            switch (c) {
```

Comment:
Can you have a comment with a link to  https://docs.oracle.com/javase/8/docs/api/java/nio/file/FileSystem.html#getPathMatcher-java.lang.String- to describe these special characters are sufficient?